core/src/test/scala/unit/kafka/server/AbstractFetcherThreadTest.scala

```diff
@@ -1333,4 +1333,93 @@ class AbstractFetcherThreadTest {
 
   }
 
+  @Test
+  def testResize(): Unit = {
```

Comment:
This test would be better placed in `AbstractFetcherManagerTest`. Could we name it `testExpandThreadPool` and also add another test `testShrinkThreadPool` to test the opposite?